The `-Wempty-body` warning in `Shutdown` function at `Noise3D` sample

The `gcc` compiler told me, when compilling `Noise3D` sample:
```shell
[ 87%] Building C object Chapter_14/Noise3D/CMakeFiles/Noise3D.dir/Noise3D.c.o
../opengles3-book/Chapter_14/Noise3D/Noise3D.c:499:38: warning: if statement has empty body [-Wempty-body]
   if ( userData->texCoords != NULL );
                                     ^
```

--- a/Chapter_14/Noise3D/Noise3D.c
+++ b/Chapter_14/Noise3D/Noise3D.c
@@ -496,8 +496,7 @@ void Shutdown ( ESContext *esContext )
       free ( userData->indices );
    }
 
-   if ( userData->texCoords != NULL );
-
+   if ( userData->texCoords != NULL )
    {
       free ( userData->texCoords );
    }

